lib/server.js

```diff
@@ -305,6 +305,13 @@ function create(log, error, config, routes, db, translator) {
     }
   )
 
+  server.ext('onPreHandler', (request, reply) => {
```

Comment:
I don't know the ins and outs of Hapi code nearly well enough, though the conversion of `features` to a Set here seems like it's far away from the logic where it's consumed. Is the intent to enable `features` for other routes? If not, can this be moved into lib/routes/sms.js somehow?